ISSUER_PROTOCOL.md

```diff
@@ -0,0 +1,333 @@
+# TrustTokenV1 Issuer Protocol
+
+This document documents the cryptographic protocol for the "TrustTokenV1" experimental version of Trust Token. An issuer needs to support maintaining a set of keys and a key commitment endpoint, as well as implementing the Issue and Redeem cryptographic functions to sign and validate Trust Tokens. Experimental versions of Trust Token are not intended to be backwards-compatible with each other and will undergo rapid design/implementation changes during the experiment timeframe.
+
+## Public Issuer Interfaces
+
+This section describes the public issuer interfaces that an issuer will need to support.
+
+### Issuer Key Commitments
+
+A Trust Token issuer should have an endpoint at a publicly accessible secure URL (HTTPS) that serves the current key commitments used in the Trust Token protocol. Request to this endpoint should result in a JSON response of the following format:
+
+
+```
+Key commitment result
```

Comment:
I think we should have one source of truth for this structure to avoid getting out of sync. 

If we have the definition here, could you please delete the definition in the design doc and add a cross-reference pointing here? Alternatively, we could update the definition in the design doc and add a cross-reference from here to there.

Reply:
The Chrome design doc isn't canonical for the API, just how Chrome implements it, so probably worth having that point at this, with the caveat that the design doc should probably pin at whatever Chrome implements, rather than the 'latest' issuer protocol.